.babelrc

```diff
@@ -1,8 +0,0 @@
-{
-  "presets": ["env", "react"],
-  "plugins": [
-    "transform-object-rest-spread",
-    "transform-react-jsx",
-    "transform-class-properties"
-  ]
-}
```

Comment:
`.babelrc` is not needed anymore.

I found out because it was causing an error, and I came across this advice that said to delete it:

https://stackoverflow.com/a/66856855/782045